Source/FluentSpreadsheets/TableScaling/ColumnTableScaler.cs

```diff
@@ -0,0 +1,27 @@
+namespace FluentSpreadsheets.TableScaling;
+
+internal class ColumnTableScaler : Scaler
+{
+    private static readonly Lazy<ColumnTableScaler> LazyInstance = new Lazy<ColumnTableScaler>(() => new ColumnTableScaler());
+
+    private ColumnTableScaler() { }
+    
+    public static ColumnTableScaler Instance => LazyInstance.Value;
+
+    protected override void ValidateCustomization(IComponent component, IComponent customizedComponent)
+    {
+        if (component.Size.Height.Equals(customizedComponent.Size.Height))
+            return;
+        
+        throw new InvalidOperationException("Customized component height must be equal to the component height.");
```

Comment:
I think exception should go into invert if and message into constant

Reply:
It is kinda arguable. First of all, if there is a possibility not to negate the condition, I'd go for it, because it reads better. And then you can interpret Validate method as a method that throws if validation wasn't successful, so it's normal to have a throw statement in "main body" of the method.